x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -211,30 +201,22 @@ DECIMAL_VALUE
     ;
 
 IDENTIFIER
-    : (LETTER | '_') (LETTER | DIGIT | '_' | '@' )*
-    ;
-
-DIGIT_IDENTIFIER
-    : DIGIT (LETTER | DIGIT | '_' | '@')+
+    : (LETTER | '_') (LETTER | DIGIT | '_')*
     ;
 
-QUOTED_IDENTIFIER
-    : '"' ( ~'"' | '""' )* '"'
-    ;
-   
 fragment EXPONENT
-    : 'E' [+-]? DIGIT+
+    : [Ee] [+-]? DIGIT+
```

Comment:
There's no need to define the lowercase since the parser automatically handles that.

Reply:
It did, but sense all of the tokens in EQL are case-sensitive (I removed the case insensitive lexer), I had to explicitly make this case-insensitive.